spec.bs

```diff
@@ -4004,64 +4041,96 @@ from querying the server during an auction.
 </div>
 
 <div algorithm>
- To <dfn>compute the key hash of ad</dfn> given an [=interest group=] |ig| and an [=interest group ad=] |igAd|:
-   1. Let |keyString| be the [=k-anonymity key=] formed from the [=string/concatenation=] of the following strings separated with U+000A LF:
+  To <dfn>compute the key hash of ad</dfn> given an [=interest group=] |ig| and an [=interest group ad=] |igAd|:
+    1. Let |keyString| be the [=k-anonymity key=] formed from the [=string/concatenation=] of the
+      following strings separated with U+000A LF:
       * "AdBid"
       * the [=serialization of an origin|serialization=] of |ig|'s [=interest group/owner=]
       * the [=URL serializer|serialization=] of |ig|'s [=interest group/bidding url=]
       * the [=URL serializer|serialization=] of |igAd|'s [=interest group ad/render url=].
-   1. Return the [=SHA-256=] hash of the [=ASCII encoding=] of |keyString|.
+    1. Return the [=SHA-256=] hash of the [=ASCII encoding=] of |keyString|.
 </div>
 
 <div algorithm>
-  To <dfn>compute the key hash of reporting ID</dfn> given an [=interest group=] |ig| and an
-  [=interest group ad=] |igAd|:
-    1. Let |keyString| be a [=k-anonymity key=] formed from the [=string/concatenation=] of the following strings separated with U+000A (LF):
+  To <dfn>compute the key part for one of multiple reporting ids</dfn> given [=string=]-or-null |reportingId|:
+    1. If |reportingId| is null, then return the [=string/concatenation=] of the following:
+      * U+0000 (NUL)

```

Comment:
There should be five of these, as per [the implementation](https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/common/interest_group/interest_group.cc;drc=e66b343b5554785a32ad988bfe5c3c524f5e1857;l=101).